src/Controller/EnMarche/AdherentController.php

```diff
@@ -239,4 +243,38 @@ public function listMyCommitteesAction(): Response
             'committees' => $manager->getAdherentCommittees($this->getUser()),
         ]);
     }
+
+    /**
+     * @Route("/mon-compte/desadherer", name="app_adherent_terminate_membership")
+     * @Method("GET|POST")
+     * @Security("is_granted('UNREGISTER')")
+     */
+    public function terminateMembershipAction(Request $request): Response
+    {
+        $adherent = $this->getUser();
+
+        $form = $this->createForm(FormType::class, null, [
+            'csrf_token_id' => self::UNREGISTER_TOKEN,
+        ]);
+
+        $form->add('word', UnregisterType::class);
```

Comment:
This should be done in the `UnregisterType`, why making it a filed instead of a form?